rest/model/test.go

```diff
@@ -53,6 +55,29 @@ func (at *APITest) BuildFromService(st interface{}) error {
 			LogId:   ToStringPtr(v.LogID),
 			LineNum: v.LineNum,
 		}
+
+		emptyStr := ""
+		isEmptyLogID := len(v.LogID) == 0
+		isEmptyURL := len(v.URL) == 0
+		isEmptyURLRaw := len(v.URLRaw) == 0
+
+		if !isEmptyURL {
+			at.Logs.HTMLDisplayURL = at.Logs.URL
+		} else if isEmptyLogID {
+			at.Logs.HTMLDisplayURL = &emptyStr
```

Comment:
It would make more sense for these to be null rather than the empty string right?